config/locales/devise.en.yml

```diff
@@ -21,6 +21,13 @@ en:
         subject: "Reset password instructions"
       unlock_instructions:
         subject: "Unlock instructions"
+      welcome: "Dear %{email}, welcome to TeSS!"
+      your_account_locked: "Your account on TeSS has been locked due to an excessive number of unsuccessful log in attempts."
+      click_link: "Click the link below to unlock your account:"
+      password_change_request: "Someone has requested a link to change your password on TeSS. You can do this through the link below."
+      ignore_if_not_you: "If you didn't request this, please ignore this email."
+      password_change_waits: "Your password won't change until you access the link above and create a new one."
+      confirm_through_link: "You can confirm your TeSS account email through the link below:"
```

Comment:
I think it would be better if the keys were under the corresponding key for each email.

e.g. `confirm_through_link` under `confirmation_instructions`.

And maybe the `hello` could be directly under `mailer` (even if it duplicates what is in `devise_invitable.en.yml`)